src/ui/pages/SendTx/index.tsx

```diff
@@ -48,44 +55,50 @@ export default function SendTx(): ReactElement {
       type: MessageTypes.CREATE_SEND,
       payload: {
         rate: +toDollaryDoos(fee),
-        outputs: [{
-          value: +toDollaryDoos(0.1),
-          address: 'hs1q7q3h4chglps004u3yn79z0cp9ed24rfr5ka9n5',
-        }],
-      }
+        outputs: [
+          {
+            value: +toDollaryDoos(0.1),
+            address: "hs1q7q3h4chglps004u3yn79z0cp9ed24rfr5ka9n5",
+          },
+        ],
+      },
```

Comment:
I know it's not a change made in this PR, but any idea what this hard-coded address is, and who owns it?

Reply:
No idea...